### DISCUSSION TITLE
Discord server????

### DISCUSSION THREAD

**Discussion by H1Mohamed** [pocketbase/pocketbase#3953] [Q&A]
Is there is a discord support server???
if not can **We** create a server for announcing updates and helping people so that this GitHub will be mostly about fixing bugs and feature reqquests
and heck its not that hard to moderate nor create a discord server thanks to auto mod and bots like mee6

**Comment by ganigeorgiev** [comment]
This was asked several times already and I'd appreciate if users stop spamming me with this.

See https://github.com/pocketbase/pocketbase/discussions/3860, https://github.com/pocketbase/pocketbase/discussions/548, https://github.com/pocketbase/pocketbase/issues/185, https://github.com/pocketbase/pocketbase/issues/471 and probably others...
